### PR TITLE
Also ignore URL for Notion and Spire in resource update

### PR DESCRIPTION
**Pull request by janelletavares**
## Description of change

Fixes <GitHub Issue>

## Type of change

- [ ]  New feature
- [x]  Bug fix
- [ ]  Refactor
- [ ]  Documentation

## How was this tested?

- [ ]  Unit Tests
- [ ]  Tested in staging
- [x]  Tested in minikube

## Demo

| before | after |
|--------|-------|
|||
Screenshot from 2023-04-11 21-30-12

Screenshot from 2023-04-11 21-29-02

### CODE REVIEW

File: cmd/meroxa/root/resources/create.go [hunk 1]
```diff
@@ -293,17 +293,20 @@ func (c *Create) handlePrivateKeyFlags(ctx context.Context) error {
 	return nil
 }
 
+const defaultNotionURL = "https://api.notion.com"
+const defaultSpireMaritimeAisURL = "https://api.spire.com/graphql"
```
[the remark this answers is not in the document]
Comment (janelletavares) [reply cmd/meroxa/root/resources/create.go:297]: i'm not sure what you mean. move where?

[the remark this answers is not in the document]
Comment (jayjayjpg) [reply cmd/meroxa/root/resources/create.go:297]: @janelletavares Does the repo maybe have a dedicated `constants` file to list hard-coded values in?

At least in `platform-ui-v1` we have something similar to make it easier to update hard-coded values in the future (as in, there's only one source of truth to be updated if needed, versus several instances across the codebase)

[the remark this answers is not in the document]
Comment (janelletavares) [reply cmd/meroxa/root/resources/create.go:297]: @jayjayjpg there is no such file in this repo.  I don't think such a thing would be common for golang, but I could be wrong.

[the remark this answers is not in the document]
Comment (jayjayjpg) [reply cmd/meroxa/root/resources/create.go:297]: Sounds good and thank you for the outline! In that case, I'm also not sure what's the best place to define those constants

[the remark this answers is not in the document]
Comment (samirketema) [reply cmd/meroxa/root/resources/create.go:297]: @janelletavares I think I commented this right as you committed the change, but these used to be in `vars`. I was initially just suggesting changing those to `const` as you already did. Thanks for doing this.

That said, would be nice to move these to the top of the file! but that's just a small style nit.

**Review by samirketema — APPROVED** [review]
this is great, thanks for the update!

**Pull request merged**